packages/env/test/github.test.js

```diff
@@ -21,6 +21,8 @@ describe('GitHub', () => {
     }));
 
     env = new PercyEnvironment({
+      PERCY_PARALLEL_TOTAL: '-1',
+      PERCY_PARALLEL_NONCE: 'job-id',
```

Comment:
```suggestion
      GITHUB_RUN_ID: 'job-id',
```

Reply:
This is what happens when you get excited to pick your dog up from the vet and rush commit 😆